src/frontend/komponenter/Felleskomponenter/Datalast/DataLastIkkeSuksess.tsx

```diff
@@ -2,32 +2,46 @@ import { Alert } from '@navikt/ds-react';
 import * as React from 'react';
 
 import HenterData from './HenterData';
+import ServerFeil from '../../../pages/feilsider/ServerFeil';
 import { type Ressurs, RessursStatus } from '../../../typer/ressurs';
 
 interface IProps {
     ressurser: (Ressurs<unknown> | undefined)[];
     henteBeskrivelse?: string;
+    visFeilSide?: boolean;
     spinnerStørrelse?: '2xlarge' | 'large';
 }
 
 const DataLastIkkeSuksess: React.FC<IProps> = ({
     ressurser,
     spinnerStørrelse,
     henteBeskrivelse,
+    visFeilSide,
 }) => {
     const filtrerteRessurser = ressurser.filter(r => r !== undefined);
     const ingenTilgangRessurs = filtrerteRessurser.find(
         r => r.status === RessursStatus.IkkeTilgang
     );
+    const serverFeil = filtrerteRessurser.find(r => r.status === RessursStatus.ServerFeil);
```

Comment:
Hva tenker du om destructing?
```suggestion
    const serverFeil = filtrerteRessurser.find({ status } => status === RessursStatus.ServerFeil);
```

Reply:
Ja, helt enig i det forslaget!